Pod/Classes/Views/Textview/UITextView+Extensions.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  UITextView+Extensions.swift
+//  Pods
+//
+//  Created by Derek Ostrander on 6/8/16.
+//
+//
+
+import UIKit
+
+extension PlaceholderConfigurable where Self:UITextView {
+    private var xConstraint: NSLayoutConstraint? {
+        let constraint = constraints
+            .filter({ (constraint: NSLayoutConstraint) -> Bool in
+                return constraint.firstAttribute == .Left &&
+                    constraint.secondAttribute == .Left &&
+                    (constraint.firstItem === self || constraint.secondItem === self)
```

Comment:
Are you sure this filter is catching the constraint below? From reading `placeholderLabel.leftAnchor.constraintEqualToAnchor(leftAnchor)` I'd think that either firstItem or secondItem would be placeholderLabel. -- same for the yConstraint below.